compose/common.yml

```diff
@@ -294,6 +301,21 @@ services:
     ports:
       - ${STAKE_POOL_PROJECTOR_PORT:-4002}:3000
 
+  transaction-projector:
```

Comment:
This name is too generic given all are some type of "transaction projection". Are you seeing this one take on more responsibility than deriving protocol params @iccicci?

Reply:
There are multiple reasoning about architecture here.

- Governance actions proposal projection is supposed to project a really small amount of data: adding a dedicated microservice for it doesn't sounds as a good idea.
- The projector for transactions body will project a lot of data: it probably deserves a dedicated microservice.
- Given a governance actions proposal id is the couple `txId, index`, probably the better microservice to include this projection is the one which will project transactions body.

This is the skeleton for transactions body projection microservice and I called it `transaction-projection`; would you mind to propose a more appropriate name?

Thank you

Reply:
Agree with you here @iccicci. Let's make this the container for the optimised wallet API 😎 How about `wallet-api-projector`? This is a slight departure from the very granular approach we've been taking but there really should just be one projector to prepare the data for that API IMO.

Reply:
1369e9be9e5 should solve this